src/arith_uint256.h

```diff
@@ -26,6 +26,7 @@ class base_uint
 protected:
     static_assert(BITS / 32 > 0 && BITS % 32 == 0, "Template parameter BITS must be a positive multiple of 32.");
     static constexpr int WIDTH = BITS / 32;
+    /** Number represented in least-significant-first base 2^32 digits. */
```

Comment:
I had to read this several times (didn't understand what a "least-significant-first base" is at first), consider the following:
```suggestion
    /** Integer with 32-bit digits, least-significant first. */
```